fsw/src/sample_app.h

```diff
@@ -37,18 +37,70 @@
 #include "cfe_evs.h"
 #include "cfe_sb.h"
 #include "cfe_es.h"
+#include "cfe_tbl_filedef.h"
+#include "cfe_tbl_task.h"
+#include "cfe_tbl_internal.h"
```

Comment:
Needed CFE_TBL_RegistryRec_t and CFE_TBL_LoadBuff_t from cfe_tbl_task.h 
Needed the variable CFE_TBL_NOT_FOUND in cfe_tbl_internal.

I don't think they are critical unless you want to manually do an CRC calculation instead of relying on CFS. See new commit for changes.

Reply:
cfe_tbl_internal.h is intented to be internal to tbl use only, what depends on this definition?

Reply:
I concur that the `cfe_tbl_task.h` and `cfe_tbl_internal.h` files should _not_ be required here.  These are internal to the TBL subsystem implementation and not part of its public API.  As far as I know, no other table-using CFS apps need to access these internal structures.  With the exception of elf2cfetbl, which documented as needing cleanup to remove this.